pkg/docker/config/config.go

```diff
@@ -587,25 +614,57 @@ func findAuthentication(registry, path string, legacyFormat bool) (types.DockerA
 		return getAuthFromCredHelper(ch, registry)
 	}
 
-	// I'm feeling lucky
-	if val, exists := auths.AuthConfigs[registry]; exists {
-		return decodeDockerAuth(val)
-	}
+	// Support for different paths in auth.
```

Comment:
Absolutely non-blocking:
```suggestion
	// Support for different paths in auth.
	// (This is not a feature of ~/.docker/config.json; we support it even for those files as an extension.)
```
I don’t know where to put this; I could see a case for writing this in the man page but that man page doesn’t (authoritatively) document `.docker` anyway. Yet again, just the existence of this PR might be enough already.

Reply:
Added the comment as suggested